crates/avalanche-types/Cargo.toml

```diff
@@ -121,7 +121,7 @@ default = [
     # "message",
     # "mnemonic",
     # "proto",
-    # "subnet",
+    "subnet",
```

Comment:
Why make this a default feature? If it should be default, we can just remove the feature flag altogether. 

(don't have to do this right now, I'm just curious)